benchmark/benchmark_static_ocean.jl

```diff
@@ -14,7 +14,7 @@ const timer = TimerOutput()
 Nt = 10  # Number of iterations to use for benchmarking time stepping.
 
 # Model resolutions to benchmarks. Focusing on 3D models for GPU benchmarking.
-            Ns = [(32, 32, 32), (64, 64, 64), (128, 128, 128), (256, 256, 256)]
+            Ns = [(16, 16, 16), (32, 32, 32), (64, 64, 64), (128, 128, 128), (256, 256, 256)]
```

Comment:
Is it worth adding `n_threads = [1, 2, 4, 8, 16, 32]` to this benchmark?

Reply:
Yeah I've been meaning to do a multithreading benchmark but couldn't figure out if you can change the number of threads while in Julia. If we can't then we need a bash script to spin up different Julia processes with different numbers of threads.

Either way I think it should be a separate benchmark, e.g. `benchmark_multithreading.jl, but I agree it's a much more useful benchmark.

Reply:
Can't we use `ENV["JULIA_NUM_THREADS"] = 2`?

Reply:
Hmmm, not sure if it'll work if you change it on the fly (I thought threads are initialized at Julia startup?) but I'll try.

Reply:
I suspect you're right

Reply:
```julia
julia> Threads.nthreads()
8

julia> ENV["JULIA_NUM_THREADS"]
"8"

julia> ENV["JULIA_NUM_THREADS"] = 1
1

julia> Threads.nthreads()
8
```